src/collections/_documentation/error-reporting/configuration/drain-example/javascript.md

```diff
@@ -1,12 +1,15 @@
-The client provides a close method that optionally takes the time in milliseconds for how long it
-waits and will return a promise that resolves when everything was flushed or the timeout kicked
+The `close` method optionally takes a timeout in milliseconds and returns a
+promise that resolves when all pending events were flushed or the timeout kicked
 in.
 
 ```javascript
-const client = Sentry.getCurrentHub().getClient();
-if (client) {
-  client.close(2000).then(function() {
-    process.exit();
-  });
-}
+Sentry.close(2000).then(function() {
+  process.exit();
+});
 ```
+
+After calling `close` the current client cannot be used any more so make sure to
+only do that right before you shut down the application.
```

Comment:
```suggestion
 do that before you shut down the application.
```

Reply:
Hmm, in this case here "only" was used to say "if you do it, you should only do it in this circumstance (when the app is shutting down)".

Without "only", it reads:

> After calling `close`, the current client cannot be used anymore, so make sure to do that before you shut down the application.

I may be misinterpreting, I read the above as a suggestion to *always* ("make sure") call `close` before shutting down the app. And that suggestion would be incorrect.

Shall we keep "only" and delete "right"?

Reply:
How about:

After calling `close`, the current client cannot be used anymore. It's important to only call `close` immediately before shutting down the application.

Reply:
I think since these steps are so dependent on each other, we should avoid using the word "that" and "it" and be as explicit as possible. And the word "right" is not quite precise enough, so I changed it to "immediately." Let me know if that works. If not, we can try rewriting the sentence, again.

Reply:
Makes sense. I've taken your suggestion and updated the relevant sentences in the multiple files. Thanks!